Assets/MixedRealityToolkit.Services/InputSystem/FocusProvider.cs

```diff
@@ -32,6 +32,14 @@ public class FocusProvider : BaseDataProvider, IMixedRealityFocusProvider
         private PointerHitResult hitResult3d = new PointerHitResult();
         private PointerHitResult hitResultUi = new PointerHitResult();
 
+        public ref readonly Dictionary<uint, IMixedRealityPointerMediator> PointerMediators
```

Comment:
>ref readonly Dictionary [](start = 15, length = 23)

I think this should be IReadOnlyDictionary instead. Dictionary is a reference type (i.e. class) so a ref to a dictionary object is a ref to its address. I believe that what you are returning there in C++ terms is Dictionary* const, instead of Dictionary const *.

Reply:
Actually, to be precise, I think `ref readonly Dictionary` translates to `Dictionary** const` in C++ and I don't think C# provides any way to have a `Dictionary const **` or `Dictionary const *` return type.

Reply:
I can see no option to approve with comments, so I'm approving it so it can be checked in once you do that change.

Reply:
Nth correction, I think `ref readonly Dictionary` translates to `Dictionary * const *` in C++.